src/components/EmailNotifications/EmailNotificationsForm.js

```diff
@@ -0,0 +1,302 @@
+import React, { useCallback, useState } from 'react'
+import styled from 'styled-components'
+import {
+  Button,
+  Checkbox,
+  Field,
+  GU,
+  IconCheck,
+  IconCross,
+  Link,
+  TextInput,
+  textStyle,
+  useTheme,
+} from '@aragon/ui'
+import LocalIdentityBadge from '../LocalIdentityBadge/LocalIdentityBadge'
+import { addressesEqual, transformAddresses } from '../../lib/web3-utils'
+import { validateEmail } from '../../utils/notifications-utils'
+
+import emailNotifcationIllustration from '../../../src/assets/emailNotifications.svg'
+
+function EmailNotificationsForm({
+  account,
+  compactMode,
+  isModal,
```

Comment:
If we apply https://github.com/aragon/court-dashboard/pull/305#discussion_r422471478, let's also do that here :).

Reply:
seems that we are not going to re use those components outside this flow, i would prefer to use a boolean rather than a text mode